discovery_child_development/notebooks/labelling/evaluate_taxonomy_labelling.ipynb

```diff
@@ -0,0 +1,1258 @@
+{
```

Comment:
Is there any chance you could repush this with cleared outputs? As then I think I may be able to see it in github as a jupyter notebook. 

You may need to add tiktoken the requirements file. 

I don't have this file : "inputs/data/labelling/taxonomy/output/training_validation_data_patents_openalex_LABELLED.jsonl"
So couldn't run the rest of the file.

Reply:
Thanks Jess, good point. I'm now adding a getter for this and will update the notebook accordingly.